lib/AppInfo/Application.php

```diff
@@ -41,8 +40,7 @@ public function __construct(array $urlParams = []) {
 
 		/** @var \OC\Server $server */
 		$server= $container->getServer();
-
-		/** @var \OCP\Files\External\IStoragesBackendService $backendService */
+		/* @phan-suppress-next-line PhanUndeclaredMethod */
```

Comment:
@phil-davis `getStoragesBackendService` returns ` \OCP\Files\External\IStoragesBackendService` _i.e removed here_, but phan fails! :confused:

Reply:
The complaint from phan is probably that it does not understand that the `$server` object has a method `getStoragesBackendService()`

But it is working out (somehow by itself) what `$backendService` is because it does not complain about the next lines that call methods of `$backendService`

This is OK for now. Someone in future who looks through all the `phan-suppress-next-line` in detail will have to sort this out.